.github/workflows/ci-pkg-install.yml

```diff
@@ -17,141 +17,40 @@ defaults:
 
 jobs:
 
-  init-temp:
-    runs-on: ubuntu-20.04
-    steps:
-    - run: mkdir pypi && touch pypi/.placeholder
-    - uses: actions/upload-artifact@v3
-      with:
-        name: ci-packages-${{ github.sha }}
-        path: pypi
-
-  install-standalone:
-    needs: init-temp
+  install-pkg:
     runs-on: ${{ matrix.os }}
     strategy:
-      fail-fast: true
-      max-parallel: 1
+      fail-fast: false
       matrix:
         os: [ubuntu-20.04, ubuntu-22.04, macOS-11, macOS-12, windows-2022]
```

Comment:
Why do we need multiple OS versions?

Reply:
This is not an edit of this PR :otter:

Reply:
I was just asking, didn't say you needed to modify this here.

Reply:
it was added a few weeks ago as a reaction to @williamFalcon concerns that we can't install lightning on some machines such as M1 with py3.8